subiquity/controllers/keyboard.py

```diff
@@ -29,6 +29,16 @@
 class KeyboardController(SubiquityController):
 
     autoinstall_key = model_name = "keyboard"
+    autoinstall_schema = {
+        'type': 'object',
+        'properties': {
+            'layout': {'type': 'string'},
+            'variant': {'type': 'string'},
+            'toggle': {'type': 'string'},
+            },
+        'required': ['layout'],
```

Comment:
I still kind of wish of skipping keyboard question on s390x, and just using C.UTF-8 / US? keyboard layout? I need to figure out if keyboard layout question makes sense on s390x.

Reply:
Yeah, this would probably make sense. Low priority though? File a bug or something?